libkbfs/root_metadata.go

```diff
@@ -864,7 +864,9 @@ func (md *RootMetadata) IsWriter(
 		return h.IsWriter(uid), nil
 	}
 
-	// Team membership needs to be checked with the service.
+	// Team membership needs to be checked with the service.  For a
```

Comment:
Sorry, I reject these nits.  Perfectly valid way to space sentences, no matter what gabriel says.